Mathlib/CategoryTheory/Limits/Fubini.lean

```diff
@@ -195,6 +195,50 @@ def coneOfConeUncurryIsLimit {D : DiagramOfCones F} (Q : ∀ j, IsLimit (D.obj j
     rw [← w j]
     simp
 
+/-- Conversely, if `coneOfConeUncurry Q c` is a limit cone then `c` is in fact a limit cone.
+-/
+def isLimitOfConeOfConeUncurryIsLimit {D : DiagramOfCones F} (Q : ∀ j, IsLimit (D.obj j))
```

Comment:
It seems that this would be a better name according to our naming conventions:
```suggestion
def isLimitOfConeOfIsLimitConeUncurry {D : DiagramOfCones F} (Q : ∀ j, IsLimit (D.obj j))
```
(same of the other definition)